mlflow/server/js/src/experiment-tracking/components/traces/quickstart/TraceTableQuickstart.utils.tsx

```diff
@@ -15,6 +15,72 @@ export type QUICKSTART_FLAVOR =
   | 'gemini'
   | 'custom';
 
+export const QUICKSTART_TAB_MESSAGES: Record<QUICKSTART_FLAVOR, React.ReactNode> = {
+  openai: (
+    <FormattedMessage
```

Comment:
Can we simplify further by constructing `Record<QUICKSTART_FLAVOR, { defaultMessage: string; description: sting; } >` rather than using `FormattedMessage` as the value?